data-tiles/GNUmakefile

```diff
@@ -1,52 +1,40 @@
 #
 # This makefile requires GNU make
 #
-# Important targets:
+# Run "make help" to get help about important targets and variables to set.
 #
-#	all -- do everything
+# You can set variables on the make command line, or set them in a params.mk file.
+# Use PARAMS=some-params.mk to use a specific params.mk file:
 #
-#	dirs -- create data directory structure
-#	binaries -- compile Go programs (you may need -B)
+#	make all PARAMS=some-params.mk
```

Comment:
a note here about the params files thats already exist might be helpful - I missed them at first.